frontend/src/components/Director.vue

```diff
@@ -139,6 +149,14 @@ export default {
         transition(name="slide-fade")
           p.director-listening-state(v-show="state == STATES.ACTIVE") Listening...
       a(:href="`${$router.history.current.path}/monitor`" target="_blank").go-to-monitor Go to monitor
+      a-dropdown(:trigger="['click']")
+        a.ant-dropdown-link(@click="e => e.preventDefault()") {{this.user.director.language}}
+          a-icon(type='down')
+        a-menu(slot='overlay')
+          a-menu-item(key='0')
+            a(href="#" @click="changeDirectorsLanguage('en-US')") en-US
+          a-menu-item(key='1')
+            a(href="#" @click="changeDirectorsLanguage('tr-TR')") tr-TR
```

Comment:
please add a static object that defines available languages—it should look like the following:

```js
const availableLanguages  = {
  'en-US': 'English (US),
  'tr-TR': 'Turkish'
}
```

and then you can simply loop over `Object.keys(availableLanguages)` and print the `key` and `availableLanguages[key]` inside the menu item.

This also helps you to display `this.user.director.language` in a human readable way by simply doing `availableLanguages[this.user.director.language]` above.